### PR TITLE
created dinosaur function

### PR DESCRIPTION
**Pull request by ruckeran**
please check this dinosaur function
see #20

### CODE REVIEW

**Review by nickhaf — CHANGES_REQUESTED** [review]
Works great, thanks!
Please put the code into a function, something like:
``` r
sample_dino <- function(n_sample){
sample(..., size = n_sample)
}
```

File: Dinosaur.R [hunk 1]
```diff
@@ -0,0 +1,2 @@
+x <- c("Brontosaurus", "Trex", "Triceratops")
+sample(x, 1)
```
Comment (burbliesj) [inline Dinosaur.R]: Looks good to me, but I am no expert with functions, so @nickhaf, please double ckeck!

**Review by burbliesj — COMMENTED** [review]
Good work!

**Comment by ruckeran** [comment]
Thank you very much for your ideas! I don't know how to write this as a nice and clean function - please help!

**Comment by nickhaf** [comment]
> Thank you very much for your ideas! I don't know how to write this as a nice and clean function - please help!

Sure! I would suggest writing the function like this:

```r
sample_dino <- function(dinos = c("T.rex", "Brontosaurus", "Triceratops"), n_sample = 1){
dino_sample <- sample(dinos, size = n_sample)
return(dino_sample)
}
```

This is a simple idea, where we can provide dino names, but the function takes default names if we don't. We can also tell the function, how many dinos we want to sample.
Please write this and try it with an example.

**Review by nickhaf — APPROVED** [review]
Looks great, thanks!

**Comment by nickhaf** [comment]
@burbliesj : Please approve the pull-request if you think it can be merged.

**Comment by ruckeran** [comment]
> > Thank you very much for your ideas! I don't know how to write this as a nice and clean function - please help!
> 
> Sure! I would suggest writing the function like this:
> 
> ```r
> sample_dino <- function(dinos = c("T.rex", "Brontosaurus", "Triceratops"), n_sample = 1){
> dino_sample <- sample(dinos, size = n_sample)
> return(dino_sample)
> }
> ```
> 
> This is a simple idea, where we can provide dino names, but the function takes default names if we don't. We can also tell the function, how many dinos we want to sample. Please write this and try it with an example.

Thank you very much for your help!

**Review by burbliesj — APPROVED** [review]
Just beautiful, changes are approved!

**Pull request merged**